lamindb_setup/core/upath.py

```diff
@@ -344,7 +344,7 @@ def inner(dir_path: Path, prefix: str = "", level: int = -1):
         f"{path.name} ({n_directories} sub-{directory_info} &"
         f" {n_files} files{suffix_message}): {folder_tree}"
     )
```

Comment:
Please save `n_files` as `._n_objects` and then use this to test for it comparing with `get_stat_s3...`

Reply:
Please also rename `n_files` to `n_objects` for consistency with the `Artifact` registry and other parts of the code.

Reply:
@falexwolf comments observed. 

The way I saved `n_objects` as a private attribute in `UPath` doesn't look like great code - can you please check and provide suggestions on how to dynamically create attributes in a robust manner, if relevant?

Reply:
It isn't great code but better than using regex!

We can discuss better design over time, we're struggling a bit with building on UPath for non trivial reasons.

This PR is good to merge!

Reply:
@falexwolf in my opinion modifying attributes of UPath just for a test is not appropriate.